infrastructure/aws/development-cloudfront.tf

```diff
@@ -1,7 +1,115 @@
-# ------------
-# CloudFront
-# ------------
 
 resource "aws_cloudfront_origin_access_identity" "default" {
-  comment = "CloudFront Orgin Identity"
-}
\ No newline at end of file
+  comment = "CloudFront Origin Identity"
+}
+
+resource "aws_cloudfront_distribution" "frontend_cloudfront" {
+  origin {
+    domain_name = aws_s3_bucket.frontend.bucket_regional_domain_name
+    origin_id = "S3-${aws_s3_bucket.frontend.id}"
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.default.cloudfront_access_identity_path
+    }
+  }
+  origin {
+    domain_name = "${aws_s3_bucket.frontend.bucket}.${aws_s3_bucket.frontend.website_domain}"
+    origin_id   = "S3-Website-${aws_s3_bucket.frontend.bucket}.${aws_s3_bucket.frontend.website_domain}"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "http-only"
+      origin_ssl_protocols   = [
+        "TLSv1",
+        "TLSv1.1",
+        "TLSv1.2"
+      ]
+    }
+  }
+
+  enabled             = true
+  is_ipv6_enabled     = true
+  default_root_object = "index.html"
+
+  aliases = [
+    "${var.codename-domain}-development-cloudfront"
```

Comment:
Alias je domenove jmeno. Melo by to byt ve vysledku neco pod https certifikatem (*.ceskodigital.net) napriklad zacniucit.ceskodigital.net (zaroven je potreba nastavit DNS zaznam v route53).